pages/linux/readpe.md

```diff
@@ -0,0 +1,28 @@
+# readpe
+
+> Displays information about PE files.
+> More information: <https://manpages.debian.org/testing/pev/readpe.1.en.html>.
+
+- Display all information about the PE file:
+
+`readpe {{path/to/binary.exe}}`
```

Comment:
Just out of interest: is this the same `.exe` as Windows uses for executables?

Reply:
I modified to `readpe {{path/to/executable}}` ; I realized it is not about only `.exe` files, but also `.dll` for instance.

Reply:
Then we should remove the `.exe` part. However if it's only `.exe` and `.dll`, we could add that to the description.

Reply:
Beware that it is not only about EXE and DLL's, but also OCX, SYS, SCR, DRV or EFI.
I suggest to keep it like this : `{{path/to/executable}}`.